File: messages/src/main/java/com/thundashop/core/asanamanager/AsanaTask.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.thundashop.core.asanamanager;

import java.util.ArrayList;
import java.util.Date;
import java.util.List;

/**
 *
 * @author ktonder
 */
public class AsanaTask {
    public long id;
    public String name;
    
    public Date created_at;
    public Date modified_at;
    public String notes;
    public Boolean completed;
    public String assignee_status;
    public Date completed_at;
    public String assignee;
    
    public long projectId;
    
    public List<AsanaCustomField> custom_fields = new ArrayList();
}

File: html5apps/terminal/events/core/asanamanager/AsanaTask.php
<?php
class core_asanamanager_AsanaTask {
	/** @var String */
	public $id;

	/** @var String */
	public $name;

	/** @var String */
	public $created_at;

	/** @var String */
	public $modified_at;

	/** @var String */
	public $notes;

	/** @var String */
	public $completed;

	/** @var String */
	public $assignee_status;

	/** @var String */
	public $completed_at;

	/** @var String */
	public $assignee;

	/** @var String */
	public $projectId;

	/** @var core_asanamanager_AsanaCustomField[] */
	public $custom_fields;

}
?>